db/migrate/20200116140132_rename_reply_id_column.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class RenameReplyIdColumn < ActiveRecord::Migration[6.0]
+  def up
+    Migration::ColumnDropper.mark_readonly(:post_replies, :reply_id)
+
+    add_column :post_replies, :reply_post_id, :integer
```

Comment:
I'm quite surprised that it's possible to add a non-nullable column without problems. Maybe it works because there's an `UPDATE` inside the same transaction.